### ISSUE TITLE
Possibile to make relationship schema?

### ISSUE THREAD

**Issue by SoulRaven** [n8barr/automotive-model-year-data#4]
Is possible to make this DB to be relational?
to have 2 tables, one for makers and another for model and year?
i ask this because will be more easy and will have smaller footprint because maker will not be on each model the same.

**Comment by jspizziri** [comment]
On the same note as @soulraven, I highly recommend that:
1. You normalize your data model to removed extraneous repetitions
   - I.E. a vehicle_make table, a vehicle_model table, and a vehicle_year table.
2. Rather than offer the data as a SQL scheme and insert, you offer it my something more agnostic, for instance JSON
   - Perhaps you could do a both-and, and have a script that would generate one from the other

I've submitted a PR: https://github.com/n8barr/automotive-model-year-data/pull/16

**Comment by chilion** [comment]
Maby check this one out...

https://github.com/chilion/car-make-and-model-world-wide

**Comment by jspizziri** [comment]
@chilion,

You're still locked into MySQL.

**Comment by chilion** [comment]
True, but if I see the activity on this repo I seriously doubt if your PR will be approved. You can always adjust it to fit with mine, I will approve and merge it :)

**Comment by jspizziri** [comment]
@chilion

It does look like @n8barr isn't paying attention to external activity; however the repo was updated 5mo ago. If there is no activity on the PR I'll definitely consider migrating over to your repo, although if I do that I would probably propose some more structural changes as well, as I have a few ideas on how to improve your data model.

**Comment by chilion** [comment]
Jacob, please do that, I don't have the time for it, but there indeed is a
need for some changes.

**Comment by chilion** [comment]
@jspizziri looks like there is not activity done here :(
